about/list_of_features.rst

```diff
@@ -663,6 +665,11 @@ XR support (AR and VR)
 
   - Including support for popular stand alone headsets like the Meta Quest 1/2/3 and Pro, Pico 4, Magic Leap 2, and Lynx R1.
 
+- Out of the box limited support for VisionOS Apple Headsets.
+
+  - Currently only exporting a application for use on a flat plane within the
+    headset is supported. Immersive experiences are not supported.
```

Comment:
visionOS is has a lowercase "v" as per the [official site](https://www.apple.com/os/visionos/):

```suggestion
- Out of the box limited support for visionOS Apple headsets.

  - Currently only exporting a application for use on a flat plane within the
    headset is supported. Immersive experiences are not supported.
```